### PR TITLE
NXP-26736: add dedicated jenkins pipeline file on 9.10 maintenance br…

### PR DESCRIPTION
**Pull request by guirenard**
…anch

### CODE REVIEW

File: Jenkinsfiles/web-ui-pipeline.groovy [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    parameters([
+            string(name: 'BRANCH', defaultValue: '', description: 'Branch to test, fall-backs on $BASE_BRANCH if not found.', trim: false),
+            choice(name: 'BASE_BRANCH', choices: ['9.10'], description: 'The branch to fallback on when $BRANCH is not found.'),
+            string(name: 'SLAVE', defaultValue: 'SLAVE910', description: 'Slave label to be used.', trim: false),
+            booleanParam(name: 'CLEAN', defaultValue: false, description: 'Run npm and bower cache clean?'),
+            booleanParam(name: 'SAUCE_LAB', defaultValue: true, description: 'Should unit tests be run on Sauce Lab (or just Chrome on the slave)?'),
+            booleanParam(name: 'CREATE_PR', defaultValue: true, description: 'Should PRs be created if build is successful?'),
+            booleanParam(name: 'SKIP_IT_TESTS', defaultValue: false, description: 'Should the functional tests be skipped?'),
+            booleanParam(name: 'SKIP_UNIT_TESTS', defaultValue: false, description: 'Should the element\'s tests be skipped?'),
+            booleanParam(name: 'GENERATE_METRICS', defaultValue: false, description: 'Should the metrics report be generated?'),
```
Comment (guirenard) [inline Jenkinsfiles/web-ui-pipeline.groovy]: I guess we didn't have metrics back in that time? right @Gabez0r ?

File: Jenkinsfiles/web-ui-pipeline.groovy [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+            string(name: 'BROWSER', defaultValue: 'chrome', description: 'The browser to use for functional tests.', trim: false),
```
Comment (guirenard) [inline Jenkinsfiles/web-ui-pipeline.groovy]: Same here, this option is useless?

File: Jenkinsfiles/web-ui-pipeline.groovy [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+            string(name: 'BROWSER_BINARY', defaultValue: '', description: 'The path to the browser binary.', trim: false),
```
Comment (guirenard) [inline Jenkinsfiles/web-ui-pipeline.groovy]: Sama same? @Gabez0r ?

File: Jenkinsfiles/web-ui-pipeline.groovy (hunk 2 above)
Comment (nmpcunha) [reply Jenkinsfiles/web-ui-pipeline.groovy]: I think so!

**Comment by nmpcunha** [comment]
👍

File: Jenkinsfiles/web-ui-pipeline.groovy (hunk 1 above)
Comment (Gabez0r) [reply Jenkinsfiles/web-ui-pipeline.groovy]: Right!

File: Jenkinsfiles/web-ui-pipeline.groovy (hunk 2 above)
Comment (Gabez0r) [reply Jenkinsfiles/web-ui-pipeline.groovy]: Yes!

File: Jenkinsfiles/web-ui-pipeline.groovy (hunk 3 above)
Comment (Gabez0r) [reply Jenkinsfiles/web-ui-pipeline.groovy]: [This one is available too](https://github.com/nuxeo/plugin-nuxeo-web-ui/blob/2.2_9.10/nuxeo-web-ui-ftest/wdio.conf.js#L65).

**Pull request merged**